packages/core/src/state-management/repository.ts

```diff
@@ -340,11 +340,13 @@ export class Repository {
     if (opts.pin !== undefined && opType !== OperationType.CREATE) {
       const pinStr = opts.pin ? 'pin' : 'unpin'
       const opStr = opType == OperationType.UPDATE ? 'update' : 'load'
-      throw new Error(
+      // An error should be thrown once fully deprecated
```

Comment:
```suggestion
      // TODO(CDB-2314): An error should be thrown once fully deprecated
```

Reply:
can you also add TODOs around the commented out tests?  Otherwise I'm worried about new devs going into the tests, seeing those commented-out lines and not understanding why they are there or why they are commented-out

Reply:
on second thought, maybe we should just delete those lines from the tests for now instead of leaving them sitting there commented out

Reply:
ah yeah a linear ref todo, added that 

removed the commented out test lines